db/queries/assessment_records/_helpers.py

```diff
@@ -75,7 +92,32 @@ def derive_application_values(application_json):
             "location_json_blob"
         ]
     else:
-        derived_values["location_json_blob"] = {"error": True}
+        try:
```

Comment:
This whole bit is making `derive_application_values` a really massive, hard to read function. Could we pre-fetch this and pass it in, or separate out the postcodes.io bit into a separate function - easier to read and test.

Reply:
Done